manifests/python.yml

```diff
@@ -481,6 +481,12 @@ tests/:
     test_dynamic_configuration.py:
       TestDynamicConfigV1: missing_feature
       TestDynamicConfigV2: missing_feature
+    test_sampling_delegation.py:
+      # The Python tracer does not include the "_dd.p.dm" header when it makes a
+      # sampling decision in the situation where there was no sampling decision
+      # in the extracted trace context.
+      # See <https://github.com/DataDog/system-tests/actions/runs/7530136315/job/20496229596?pr=2005>.
+      Test_Decisionless_Extraction: missing_feature
```

Comment:
You can add a brief reason why you are adding the mark missing_feature. This info will be pushed to FPD. (don't use # comments)